Source/WebKit/Shared/GPUProcessPreferencesForWebProcess.serialization.in

```diff
@@ -22,7 +22,9 @@
 
 #if ENABLE(GPU_PROCESS)
 
+header: "WebFeatures.h"
 struct WebKit::GPUProcessPreferencesForWebProcess {
+    WebKit::WebFeatures features;
```

Comment:
I think this is missing code to go to each WebPageProxy and get the enabled features on each, and also code to update it if it is updated.  See how we get the value for isWebGLEnabled()

Reply:
? `PageConfiguration::preferencesForGPUProcess()` does that conversion. We're also not updating these preferences boolean anywhere as far as I can tell. It might be a nice enhancement in the future but I don't think it's needed for the initial version.